Snowflake.Data.Tests/UnitTests/SFDbCommandTest.cs

```diff
@@ -34,6 +36,33 @@ public void TestCommandWithConnectionAndCommandText()
             Assert.AreEqual(commandText, command.CommandText);
         }
 
+        [Test]
+        public void TestCommandExecuteThrowsExceptionWhenCommandTextIsNotSet()
+        {
+            // Arrange
+            SnowflakeDbConnection conn = new SnowflakeDbConnection();
```

Comment:
the conn variable is not used, and the test passes without it

Reply:
Removed